tests/add_subdirectory/vk_layer_dispatch_table.c

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 The Khronos Group Inc.
+// Copyright 2023 Valve Corporation
+// Copyright 2023 LunarG, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+#include <vulkan/utility/vk_layer_dispatch_table.h>
+
+PFN_vkVoidFunction local_vkGetInstanceProcAddr(VkInstance instance, const char *pName) {
+    (void)instance;
+    (void)pName;
+    return nullptr;
```

Comment:
Using nullptr in a C file

Reply:
I went ahead and removed the C file test - because gtest cannot work with a C language file (can't include the gtest headers).